jobserv/git_poller.py

```diff
@@ -308,18 +333,26 @@ def _trigger(entry: PollerEntry, trigger_name: str,
     }
     p = urlparse(change_params['GIT_URL'])
     url = p.scheme + '://' + p.netloc
+    commit_skipped = False
     if url == 'https://github.com':
-        data['reason'] += '\n' + _github_log(entry.trigger, change_params)
+        commit_summary, commit_skipped = _github_log(entry.trigger, change_params)
+        data['reason'] += '\n' + commit_summary
     elif url in GITLAB_SERVERS:
-        data['reason'] += '\n' + _gitlab_log(entry.trigger, change_params)
+        commit_summary, commit_skipped = _gitlab_log(entry.trigger, change_params)
+        data['reason'] += '\n' + commit_summary
     else:
         capable = _cgit_repos.get(url, True)
         if capable:
-            reason = _cgit_log(entry.trigger, change_params)
-            if reason:
-                data['reason'] += '\n' + reason
+            commit_summary, commit_skipped = _cgit_log(entry.trigger, change_params)
+            if commit_summary:
+                data['reason'] += '\n' + commit_summary
             else:
                 _cgit_repos[url] = False
+    if commit_skipped:
+        log.info(
+            'Skipping build for %s because of skip-ci message' %
+            entry.trigger.project)
```

Comment:
Just a little python nicety with logging.*. It take string format so you can do log.info('foo %s', 'blah'). What's nice about doing it this way is that when the loglevel is "WARN", then python won't build the actual string. So its just a nice habit.

Reply:
I thought the snippet you'd shared had a typo. Turns out I didn't know that functionality of the logger.  Very nice.  Fixed.